terraform/main.tf

```diff
@@ -7,7 +7,7 @@ provider "google" {
 }
 
 resource "google_compute_network" "main" {
-  name                    = "main-network"
+  name                    = "containerssh"
   auto_create_subnetworks = false
 }
```

Comment:
reason for this change:
Calling the resource `main` in Terraform makes it easier for us to refer to, for example
```
google_compute_network.main
```
Calling it `containerssh` instead of `main-network` helps us to recognize the network more easily on google cloud.

(not sure if this is a better practice though :)

Reply:
later on if we are working with some projects, that are much bigger and each have it owns networks, i think using project name is better :D